python/py_helpers.test.py

```diff
@@ -391,6 +391,25 @@ def test_has_call(self):
         self.assertTrue(node.has_call("int('1')"))
         self.assertTrue(node.has_call("obj.foo('spam')"))
 
+    def test_func_has_call(self):
+        code_str = """
+
+srt = sorted([5, 1, 9])
+        
+def spam(lst):
+  return lst.sort()
+
+def eggs(dictionary):
+  if True:
+    k = dictionary.get(key)
+"""
+        node = Node(code_str)
+
+        self.assertTrue(node.func_has_call("spam", "sort"))
+        self.assertTrue(node.func_has_call("eggs", "get"))
+        self.assertFalse(node.func_has_call("srt", "sorted"))
```

Comment:
I might be confused by the description. Shouldn't this be caught as using `sorted` function?

Reply:
No, because it can search only within a function but `sorted(...)` is assigned to the variable `srt` (not function) in the global scope. I forgot to mention that.

Reply:
that would still be cheating with extra steps... would it be possible to prohibit that to?

Reply:
If we want that I can simply search the entire code instead of a function. What about something like this?
```py
node.block_has_call('sorted', 'function') # finds sorted within function
node.block_has_call('sorted') # finds sorted in the entire code
```

Reply:
would that find also reassignments?